src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -0,0 +1,2579 @@
+  - /docs/agents/net-agent/net-agent-api/settransactionname-net-agent
+  - /docs/agents/net-agent/net-agent-api/set-transaction-name
+  - /docs/apm/agents/net-agent/net-agent-api/settransactionname-net-agent-api
+  - /docs/agents/net-agent/net-agent-api/start-agent
+  - /docs/agents/net-agent/net-agent-api/startagent-net-agent
+  - /docs/apm/agents/net-agent/net-agent-api/start-agent
+  - /docs/agents/net-agent/net-agent-api/tracemetadata-net-agent-api-0
+  - /docs/apm/agents/net-agent/net-agent-api/tracemetadata-net-agent-api-0
+
+freshnessValidatedDate: never
+---
+
+New Relic's .NET agent includes an API that allows you to extend the agent's standard functionality. For example, you can use the .NET agent API for:
+
+* Customizing your app name
+* Creating custom transaction parameters
+* Reporting custom errors and metrics
+
+You can also customize some of the .NET agent's default behavior by adjusting [configuration settings](/docs/agents/net-agent/configuration/net-agent-configuration) or using [custom instrumentation](/docs/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation).
+
+## Requirements
+
+To use the .NET agent API, make sure you have the [latest .NET agent release](/docs/release-notes/agent-release-notes/net-release-notes). Then, add a reference to the agent in your project using one of the two options below:
+
+   * Add a reference to `NewRelic.Api.Agent.dll` to your project.
+    OR
+   * View and download the API package from the [NuGet Package Library](https://www.nuget.org/packages/NewRelic.Agent.Api/).
+
+## List of API calls
+
+   The following list contains the different calls you can make with the API, including syntax, requirements, functionality, and examples:
+
+<CollapserGroup>
+    <Collapser
+        id="DisableBrowserMonitoring"
+        title="DisableBrowserMonitoring"
+    >
+    ### Syntax
+
+        ```cs
+        NewRelic.Api.Agent.NewRelic.DisableBrowserMonitoring([boolean $override])
+        ```
+
+        Disable automatic injection of browser monitoring snippet on specific pages.
+
+    ### Requirements
+
+    * Compatible with all agent versions.
+    * Must be called inside a [transaction](/docs/glossary/glossary/#transaction).
+
+    ### Description
+
+    Add this call to disable the <DoNotTranslate>**automatic**</DoNotTranslate> injection of [<InlinePopover type="browser" />](/docs/browser/new-relic-browser/getting-started/new-relic-browser) scripts on specific pages. You can also add an optional override to disable <DoNotTranslate>**both**</DoNotTranslate> manual and automatic injection. In either case, put this API call as close as possible to the top of the view in which you want browser disabled.
+
+    <Callout variant="tip">
+    Compare [`GetBrowserTimingHeader()`](/docs/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent), which <DoNotTranslate>**adds**</DoNotTranslate> the browser script to the page.
```

Comment:
This was formerly a link to another doc, but now should presumably be an anchor link within the same doc. Would be good to check for other examples where links like this need updating.